### PR TITLE
Accessibility features Pull request

### PR DESCRIPTION
**Pull request by adamteddychang**
- Page Titles - no issues found
- Image text alternatives  - edited alt text at lines 31, 48-52, 62, and 211-215.
- [Lines 48-52](https://github.com/adamteddychang/Hello-Portfolio/blob/Accessibility-features/index.html#L48-L52)
-  [Lines 211-215](https://github.com/adamteddychang/Hello-Portfolio/blob/Accessibility-features/index.html#L211-L215)
- Text headings - edited from h1 to h2 at lines 64, 79, 89, 100, 122, and 133
- [Line 64](https://github.com/adamteddychang/Hello-Portfolio/blob/Accessibility-features/index.html#L64)
- Color contrast - no issues found
- Resize - added new media query for zoomed page, line 809 to line 842
- [ Lines 809-842](https://github.com/adamteddychang/Hello-Portfolio/blob/Accessibility-features/style.css#L809-L842)
- Interaction - no issues found
- Moving content - no issues found
- Multimedia - no issues found 
- The basic structure of the page - changed bottom div to footer, lines 209 and 217
- [Lines 209-217](https://github.com/adamteddychang/Hello-Portfolio/blob/Accessibility-features/index.html#L209-L217)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
                 <li class="nav-list-items"><a href=""><img src="Images/mail.svg" alt=""></a></li>
```
Comment (SajjadAhmad14) [inline index.html:28]: The report should include a check about page titles.
The report should include a check about image text alternatives (img alt="alt text ").
The report should include a check about text headings.
The report should include a check about color contrast.
The report should include a check about resizing.
The report should include a check about interaction.
The report should include a check about moving content.
The report should include a check about multimedia.
The report should include a check about the basic structure of the page.

Please include a statement  about every point checked in the result section of the report as described below:

If everything was ok, please make a statement about that (e.g. "Page titles - no issues found").
If issues were found, please provide a link to the corresponding line of code in the Pull Request which you change to follow the accessibility rule.

Please go through the requirements of this milestone from [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md) link.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏
### Please write a detailed accessibility report of your project and add it to the description of the pull request.
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (adamteddychang) [reply index.html:28]: I have edited the pull request, thank you for the review!

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**